kudo-operator/params.yaml

```diff
@@ -47,9 +47,14 @@ enableMetrics:
   displayName: "Enable metrics"
 
 metricsPort:
```

Comment:
the naming is not consistent with `appMetricsPort`. What if we rename it to `operatorMetricsPort`?

Reply:
done